### DISCUSSION TITLE
Readthedocs badge doesn't appear on Phone

### DISCUSSION THREAD

**Discussion by mendax1234** [pradyunsg/furo#474] [Potential Issue] [answered]
l try using my phone to view the documentation built on this theme , but l find that the readthedocs badge disappearing. l think it is a bug since sphinx_rtd_theme doesn't have this bug. Hope this bug can be fixed! @pradyunsg 

And this is what l call readthedocs badge:
![image](https://user-images.githubusercontent.com/62934294/177001636-8d6403e9-6bb6-4f53-ac1d-d84fb641f2d3.png)

**Comment by pradyunsg** [comment]
It's in the sidebar?

**Reply by mendax1234** [reply]
> It's in the sidebar?

l have checked it on my iphone , it isn't in the sidebar.

**Comment by pradyunsg** [comment, marked as answer]
Please share a reproducer/link for this, so that I can see the behaviour that you are seeing.

**Reply by pradyunsg** [reply]
And, there's no need to @mention me in every discussion that you file.

**Reply by mendax1234** [reply]
> Please share a reproducer/link for this, so that I can see the behaviour that you are seeing.

Sure, open this [link](https://6940swerve-docs.readthedocs.io/en/latest/) in your phone and check where the readthedocs is.

**Reply by pradyunsg** [reply]
![Screenshot_20220702-151249_Firefox](https://user-images.githubusercontent.com/3275593/177004405-56048150-c2b4-441d-b5f1-91b539c04d04.jpg)

Seems fine to me.

**Reply by mendax1234** [reply]
> ![Screenshot_20220702-151249_Firefox](https://user-images.githubusercontent.com/3275593/177004405-56048150-c2b4-441d-b5f1-91b539c04d04.jpg)
> 
> Seems fine to me.

Oh , safari's bottom bar blocks this, so l can't see it. Sorry for the inconvenience 😂
